packages/ui/native.ts

```diff
@@ -11,3 +11,6 @@ export { TouchableOpacity } from './src/components/button/touchable-opacity.nati
 export type { Theme } from './src/theme-native';
 export { Icon } from './src/icons/icon/icon.native';
 export * from './src/icons/index.native';
+export { Flag } from './src/components/flag/flag.native';
+export { ItemLayout } from './src/components/item-layout/item-layout.native';
```

Comment:
I had forgotten to include `ItemLayout` + `Flag` here so they weren't available in the public package